node-graphql-client/src/graphql-js-client.js

```diff
@@ -1,14 +1,19 @@
-import fetch from 'node-fetch';
-import Client from 'graphql-js-client';
-import typeBundle from './types';
+import GraphQLClient from 'graphql-js-client';
+
+// This is the generated type bundle from graphql-js-schema
+import types from './types';
+
+const fetch = require('node-fetch');
 
 global.fetch = fetch;
 
-export default new Client(typeBundle, {
+const client = new GraphQLClient(types, {
   url: 'https://graphql.myshopify.com/api/graphql',
   fetcherOptions: {
     headers: {
       'X-Shopify-Storefront-Access-Token': 'dd4d4dc146542ba7763305d71d1b3d38'
     }
   }
 });
+
+module.exports = client;
```

Comment:
Why did this have to change from a default export?

Reply:
Tracked this down to https://babeljs.io/docs/en/v7-migration-api#export-changes. Was able to revert back to `export default` but required a change to the way `require` is done.